internal/tag/tag_test.go

```diff
@@ -3,28 +3,30 @@ package tag
 import (
 	"testing"
 
-	"github.com/auth0/go-auth0/internal/testing/expect"
+	"github.com/stretchr/testify/assert"
 )
 
-type test struct {
+type testCaseTag struct {
 	Foo bool  `scope:"foo"`
 	Bar *bool `scope:"bar"`
 	Baz *bool `scope:"baz"`
 	Bam *bool `scope:"bam"`
```

Comment:
Is this field necessary, given that it's not being used in the test?

Reply:
Good catch 👍🏻  Removed in [6e84871](https://github.com/auth0/go-auth0/pull/63/commits/6e84871ab18d4f0475d6d1495c65903248e1e10e)